ui/package.json

```diff
@@ -13,11 +13,10 @@
     "react-dom": "^16.3.0",
     "react-scripts": "^2.0.0",
     "react-select": "^2.0.0",
-    "react-vega": "^4.0.2",
     "react-vega-lite": "^2.0.2",
-    "vega-embed": "^3.29.0",
     "vega-lib": "^4.4.0",
-    "vega-lite": "^3.0.0-rc12"
+    "vega-lite": "^3.0.0-rc12",
```

Comment:
Add a comment for why these two are here, so someone else in the future doesn't make the same mistake. See https://stackoverflow.com/a/14221781/6447189

Reply:
Done.